app/models/manageiq/providers/lenovo/physical_infra_manager/physical_server.rb

```diff
@@ -1,5 +1,7 @@
 module ManageIQ::Providers
   class Lenovo::PhysicalInfraManager::PhysicalServer < ::PhysicalServer
+    has_one :hardware
```

Comment:
This isn't needed here, as this `Lenovo::PhysicalInfraManager::PhysicalServer` inherits from `PhysicalServer` in the main ManageIQ repo. That model has the dependencies. See this PR https://github.com/ManageIQ/manageiq/pull/14430